test/main.cpp

```diff
@@ -159,7 +271,7 @@ TEST_CASE("Manipulate cursor position through character writing",
             static_cast<uint8_t>(r2d2::claimed_display_cursor::OPEN_CURSOR));
 
         REQUIRE(cursor_post.cursor_x ==
-                (start_x + ((r2d2::display::st7735_128x160_s::width - start_x) / 8) * 8));
+                (start_x + ((r2d2::display::st7735_128x160_s::width - start_x) / 8) * 8));//124
```

Comment:
What does the 124 in the comment refer to?

Reply:
this is the result of the equation in fornt of it this is documented on the wiki aswell since if you calculate it on a "normal calculator it wil result in 128 but due to int casting or somting(was made by other team) it results in the correct 124 value, it was mainly for my own understanding

Reply:
Could you clarify that in the comment? That is good information to have